tests/unit/__init__.py

```diff
@@ -16,6 +15,8 @@
 os.environ['DYNAMO_CACHE_TABLE_NAME'] = f"dcp-matrix-service-cache-table-{os.environ['DEPLOYMENT_STAGE']}"
 os.environ['S3_RESULTS_BUCKET'] = f"dcp-matrix-service-results-{os.environ['DEPLOYMENT_STAGE']}"
 os.environ['DYNAMO_LOCK_TABLE_NAME'] = f"dcp-matrix-service-lock-table-{os.environ['DEPLOYMENT_STAGE']}"
+os.environ['BATCH_CONVERTER_JOB_QUEUE_ARN'] = f"test-job-queue"
+os.environ['BATCH_CONVERTER_JOB_DEFINITION_ARN'] = f"test-job-definition"
```

Comment:
I was considering the removing dependency on environment variables here (by requiring `source environment` to be run before running unit tests), however I think it makes sense that unit tests are decoupled from the deployed state and that this is fine. Following this, I would opt to update these to "dummy" values (in another PR).